contrib_bots/lib/John/docs.md

```diff
@@ -0,0 +1,26 @@
+John
+
+With the mission of humanizing bot interactions, John aims to be your
+virtual assistant at the hour of asking for help in Zulip. John is an
+interactive bot that uses machine learning heuristics to simulate a
+conversation with the user. He have a great sense of humor and
```

Comment:
Noticed a small grammar issue. "He have" should be "He has". The subject "he" is singular, while the verb "have" is plural.